src/components/Clock.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  clockName: string,
+};
+
+export class Clock extends React.Component<Props, Readonly<State> > {
+  state = {
+    today: new Date(),
+  };
+
+  timerClockId = 0;
+
+  componentDidMount() {
+    this.timerClockId = window.setInterval(() => {
+      this.setState({
+        today: new Date(),
```

Comment:
This date will be used in `console.info`, let's save it to constant and use it in both state and `console.info`